src/src/AdminBundle/Resources/views/base.html.twig

```diff
@@ -7,59 +7,48 @@
-    {#todo: <link href="custom.css" rel="stylesheet">#}
+
+    {# todo: move to assets! #}
+    <style>
+        body {
+            padding-top: 5rem;
+        }
+        .starter-template {
+            padding: 3rem 1.5rem;
+            text-align: center;
+        }
+        form label {
+            text-align: right;
+        }
+        .tab-pane {
+            margin-top: 20px;
+        }
+    </style>
 </head>
 <body>
-<nav class="navbar navbar-expand-md navbar-dark bg-dark fixed-top">
-    <a class="navbar-brand" href="#">Event Manager</a>
-    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
-        <span class="navbar-toggler-icon"></span>
-    </button>
-    <div class="collapse navbar-collapse" id="navbarsExampleDefault">
-        <ul class="navbar-nav mr-auto">
-            <li class="nav-item">
-                <a class="nav-link" href="#">Events</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link" href="{{ path('organizers') }}">Organizers</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link" href="#">Languages</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link" href="#">Mailing list</a>
-            </li>
-            <li class="nav-item">
-                <a class="nav-link" href="{{ path('admins') }}">Admins</a>
-            </li>
-        </ul>
-        <ul class="navbar-nav">
-            <li class="nav-item">
-                <a class="nav-link" href="{{ path('logout') }}">Logout</a>
-            </li>
-        </ul>
-    </div>
-</nav>
-<main role="main" class="container content">
-    <main role="main" class="container">
+    {% block navigation %}
+    {% endblock %}
+    <main role="main" class="container content">
         {% block content %}
         {% endblock %}
+        {% block modals %}
+        {% endblock %}
+    </main>
 
-    </main><!-- /.container -->
-    {% block modals %}
-    {% endblock %}
-</main>
     <!-- Bootstrap core JavaScript
     ================================================== -->
     <!-- Placed at the end of the document so the pages load faster -->
-<script src="https://code.jquery.com/jquery-3.1.1.slim.min.js" integrity="sha384-A7FZj7v+d/sdmMqp/nOQwliLvUsJfDHW+k9Omg/a/EheAdgtzNs3hpfag6Ed950n" crossorigin="anonymous"></script>
-<script src="https://cdnjs.cloudflare.com/ajax/libs/tether/1.4.0/js/tether.min.js" integrity="sha384-DztdAPBWPRXSA/3eYEEUWrWCy7G5KFbe8fFjk5JAIxUYHKkDx6Qin1DkWx51bBrb" crossorigin="anonymous"></script>
-<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/js/bootstrap.min.js" integrity="sha384-vBWWzlZJ8ea9aCX4pEW3rVHjgjt7zpkNpZk+02D9phzyeVkE+jo0ieGizqPLForn" crossorigin="anonymous"></script>
+    <script src="https://code.jquery.com/jquery-3.1.1.slim.min.js"
```

Comment:
nebude v tomto pr aj to, ze styly budeme tahat z lokalneho file-u?

Reply:
este stale neviem ako funguju assety a povedal som si, ze to mozme nechat na neskor. Styly momentalne su ulozene v `base.html` takze sa aplikuju na vsetky podstranky a velmi lahko sa budu dat vytiahnut do samostatneho file-u ked to budeme odovzdavat.

TJ. Ano, mohlo (a mozno aj malo) by to byt uz spravene, ale zatial som to nechal tak